packages/cli/src/active-executions.ts

```diff
@@ -215,6 +215,8 @@ export class ActiveExecutions {
 			executionIds.forEach((executionId) => this.stopExecution(executionId));
 		}
 
+		if (config.getEnv('executions.mode') === 'queue') return;
```

Comment:
we have quite a few checks on `config.getEnv('executions.mode')` in this file. Maybe in the long run we should split the `shutdown` method into two separate methods instead for better readability and testability.

Reply:
For now no longer applicable, but generally I agree this method has been accruing checks over time and needs refactoring.